source/dotnet/Library/AdaptiveCards.Rendering.Wpf/AdaptiveActionRenderer.cs

```diff
@@ -31,20 +31,17 @@ public static Button CreateActionButton(AdaptiveAction action, AdaptiveRenderCon
                 Style = context.GetStyle($"Adaptive.{action.Type}"),
             };
 
-            if(action.Sentiment == AdaptiveSentiment.Positive || action.Sentiment == AdaptiveSentiment.Destructive)
+            if(String.IsNullOrWhiteSpace(action.Sentiment))
```

Comment:
Shouldn't this be negated to be `if(!String.IsNullOrWhiteSpace(action.Sentiment))`?